koku/masu/processor/ocp/ocp_report_parquet_processor.py

```diff
@@ -76,6 +80,17 @@ def create_bill(self, bill_date):
         cluster_id = utils.get_cluster_id_from_provider(provider.uuid)
         cluster_alias = utils.get_cluster_alias_from_cluster_id(cluster_id)
 
+        LOG.info(
+            log_json(
+                msg="getting or creating bill",
+                cluster_id=cluster_id,
+                cluster_alias=cluster_alias,
+                provider_uuid=provider.uuid,
+                provider_name=provider.name,
+                provider_type=provider.type,
+                customer=provider.customer,
```

Comment:
instead of customer, let's just use the schema
```suggestion
                schema=self._schema_name,
```